intra_process_demo/include/image_pipeline/watermark_node.hpp

```diff
@@ -26,11 +26,16 @@
 
 #include "common.hpp"
 
-// Node that receives an image, adds some text as a watermark, and publishes it again.
-class WatermarkNode : public rclcpp::Node
+/// Node that receives an image, adds some text as a watermark, and publishes it again.
+class WatermarkNode final : public rclcpp::Node
 {
 public:
-  WatermarkNode(
+  /// \brief Construct a WatermarkNode that accepts an image, adds a watermark, and republishes it
+  /// \param input The topic to listen on
```

Comment:
@clalancette nit^2:
```suggestion
  /// \param input The name of the topic to subscribe to
```
We tend to use `topic` and `topic name` interchangeably elsewhere too, so it's not a strong opinion.

Reply:
Fixed in 604bfcf